docs/source/references/api_spec.md

```diff
@@ -225,30 +243,47 @@ Also returns current status of peer in json string:
 **Expects**: -
 
 **Responses**:
-- 200 OK reports status:
-  + Number of connected peers, except for the reporting peer itself
-  + Number of committed blocks (block height)
-  + Total number of accepted transactions
-  + Total number of rejected transactions
-  + Uptime with nanosecond precision since creation of the genesis block
-  + Number of view changes in the current round
 
-```json
+200 OK reports status:
+
+```json5
```

Comment:
This is **not** what you would receive as a response, because we don't provide JSON5 formatted replies. This either needs to be clarified or removed.

Reply:
I'd rather clarify than remove comments

Reply:
Clarified.